apps/daimo-mobile/src/view/screen/QRScreen.tsx

```diff
@@ -155,6 +155,14 @@ export function QRCodeBox({
 // Parse QR codes from Daimo or other wallets
 // Works around potential deep linking / AASA bugs by using direct links only
 function parseQRData(data: string) {
+  console.log(`[SCAN] parsing QR data '${data}'`);
+  if (data.startsWith("ethereum:")) {
+    data = data.slice("ethereum:".length).split("@")[0];
```

Comment:
this is a new one, curious where is this from / what's the format? 👀

Reply:
ya it's wack

see https://github.com/status-im/status-mobile/issues/19986#issue-2290795999